keras_hub/api/layers/__init__.py

```diff
@@ -5,11 +5,16 @@
 """
 
 from keras_hub.src.layers.modeling.alibi_bias import AlibiBias
+from keras_hub.src.layers.modeling.anchor_generator import AnchorGenerator
+from keras_hub.src.layers.modeling.box_matcher import BoxMatcher
 from keras_hub.src.layers.modeling.cached_multi_head_attention import (
     CachedMultiHeadAttention,
 )
 from keras_hub.src.layers.modeling.f_net_encoder import FNetEncoder
 from keras_hub.src.layers.modeling.masked_lm_head import MaskedLMHead
+from keras_hub.src.layers.modeling.non_max_supression import (
+    NonMaxSuppression as NonMaxSupression,
```

Comment:
There's a typo here it looks like. I think NonMaxSuppression is the correct one right? Fix on the API export call, not here, and regenerate.

Reply:
Done